web-ng/src/app/components/feature-panel/feature-panel.component.ts

```diff
@@ -70,15 +72,16 @@ export class FeaturePanelComponent implements OnInit, OnDestroy {
             )
         )
       );
-    this.layer$ = projectService
-      .getActiveProject$()
-      .pipe(
-        switchMap(project =>
-          featureService
-            .getSelectedFeature$()
-            .pipe(map(feature => project.layers.get(feature.layerId)!))
+    this.layer$ = projectService.getActiveProject$().pipe(
+      switchMap(project =>
+        featureService.getSelectedFeature$().pipe(
+          map(feature => {
+            this.feature = feature;
```

Comment:
Seems surprising the mix side-effects with layerId this way. Feature is async, so should be treated as such as well.